src/services/utils.ts

```diff
@@ -3,6 +3,51 @@
  *
  * @author Katia Aresti
  */
+export enum KeyContentType {
+  StringContentType = 'java.lang.String',
+  DoubleContentType = 'java.lang.Double',
+  IntegerContentType = 'java.lang.Integer',
+  LongContentType = 'java.lang.Long',
+  BytesType = 'Bytes'
+}
+
+export enum ValueContentType {
+  StringContentType = 'java.lang.String',
+  DoubleContentType = 'java.lang.Double',
+  IntegerContentType = 'java.lang.Integer',
+  LongContentType = 'java.lang.Long',
+  BytesType = 'Bytes'
+}
+
+export enum Flags {
+  CACHE_MODE_LOCAL = 'CACHE_MODE_LOCAL',
+  COMMAND_RETRY = 'COMMAND_RETRY',
+  FAIL_SILENTLY = 'FAIL_SILENTLY',
+  FORCE_ASYNCHRONOUS = 'FORCE_ASYNCHRONOUS',
+  FORCE_SYNCHRONOUS = 'FORCE_SYNCHRONOUS',
+  FORCE_WRITE_LOCK = 'FORCE_WRITE_LOCK',
+  IGNORE_RETURN_VALUES = 'IGNORE_RETURN_VALUES',
+  IGNORE_TRANSACTION = 'IGNORE_TRANSACTION',
+  PUT_FOR_EXTERNAL_READ = 'PUT_FOR_EXTERNAL_READ',
+  PUT_FOR_STATE_TRANSFER = 'PUT_FOR_STATE_TRANSFER',
```

Comment:
some flags are internal only and, IMO, should be removed.
`PUT_FOR_STATE_TRANSFER`
`PUT_FOR_X_SITE_STATE_TRANSFER`
`ROLLING_UPGRADE`
`COMMAND_RETRY`

Reply:
ok